### DISCUSSION TITLE
Saludos compañeros!

### DISCUSSION THREAD

**Discussion by NoelTejeda** [community/community#56130] [GitHub Education]
### Hi everyone,

Hola espero estén todos bien, es un honor compartir con ustedes este espacio, y aprender. desde que me inicié en este mundo de la programación he visto como el tiempo que he invertido ha sido de total agrado. Tanto que he querido compartir algunas cosas básicas de mi aprendiza en mi página de github. bueno estaré al tanto de todas las ventajas que me ha ofrecido Github Education Community. Saludos!!

**Comment by samus-aran** [comment]
👋 Welcome to the GitHub Community on Discussions, @NoelTejeda  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be removed within 24-48 hours to keep our community nice and tidy 🧹.
